tests/regression/test_mesh_generation.py

```diff
@@ -54,6 +54,96 @@ def test_unit_cube():
     assert abs(integrate_one(UnitCubeMesh(3, 3, 3)) - 1) < 1e-3
 
 
+def run_one_element_advection():
+    nx = 20
+    m = PeriodicRectangleMesh(nx, 1, 1.0, 1.0, quadrilateral=True)
+    nlayers = 20
+    mesh = ExtrudedMesh(m, nlayers, 1.0/nlayers)
+    Vdg = FunctionSpace(mesh, "DG", 1)
+    Vu = VectorFunctionSpace(mesh, "DG", 1)
+    q0 = Function(Vdg).interpolate(Expression("cos(2*pi*x[0])*cos(pi*x[2])"))
+    q_init = Function(Vdg).assign(q0)
+    dq1 = Function(Vdg)
+    q1 = Function(Vdg)
+    Dt = 0.01
+    dt = Constant(Dt)
+    # Mesh-related functions
+    n = FacetNormal(mesh)
+    u0 = Function(Vu).interpolate(Expression(("1.0", "0.0", "0.0")))
+    # ( dot(v, n) + |dot(v, n)| )/2.0
+    un = 0.5*(dot(u0, n) + abs(dot(u0, n)))
+    # advection equation
+    q = TrialFunction(Vdg)
+    p = TestFunction(Vdg)
+    a_mass = p*q*dx
+    a_int = (dot(grad(p), -u0*q))*dx
+    a_flux = (dot(jump(p), un('+')*q('+') - un('-')*q('-')))*(dS_v + dS_h)
+    arhs = a_mass-dt*(a_int + a_flux)
+    q_problem = LinearVariationalProblem(a_mass, action(arhs, q1), dq1)
+    q_solver = LinearVariationalSolver(q_problem,
+                                       solver_parameters={
+                                           'ksp_type': 'preonly',
+                                           'pc_type': 'lu'
```

Comment:
Travis doesn't build with mumps, so this test fails in parallel.  But it's just a DG mass matrix, so use
```python
'pc_type': 'bjacobi',
'sub_pc_type': 'lu'
```